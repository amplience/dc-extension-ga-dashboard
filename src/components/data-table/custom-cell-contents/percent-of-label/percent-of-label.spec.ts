import { render } from '@testing-library/svelte';
import PercentOfLabel from './percent-of-label.svelte';

describe('PercentOfLabel', () => {
  it('should render a PercentOfLabel component', () => {
    const { container } = render(PercentOfLabel, { value: 25, total: 50 });

    expect(container.firstChild).toMatchSnapshot();
  });
});
