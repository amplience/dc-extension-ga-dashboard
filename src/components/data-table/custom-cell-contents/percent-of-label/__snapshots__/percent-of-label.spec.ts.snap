// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PercentOfLabel should render a PercentOfLabel component 1`] = `
<div>
  <div
    class="svelte-1av7bio"
  >
    <div
      class="percent-bar svelte-1av7bio"
    >
      <div
        class="percent-bar-marker svelte-1av7bio"
        style="width: 50%;"
      />
       
      <div
        class="percent-value svelte-1av7bio"
      >
        50%
      </div>
    </div>
  </div>
</div>
`;
